### DISCUSSION TITLE
Welcome to hymnal Discussions!

### DISCUSSION THREAD

**Discussion by dons20** [dons20/hymnal#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Xhinh** [comment]
Hello,
How may I format my local songs to the json format you used ... The data entry form would be very helpful

**Reply by dons20** [reply]
Hi, I actually don't have a special form to add new songs for this project, but it would follow the syntax of the file at `src/data/songs.ts`.

A data entry form would be useful if you wanted to add new songs, but for the scope of this project, it wasn't considered. Hope this helps!
